llvm/lib/Target/RISCV/RISCVTargetMachine.cpp

```diff
@@ -51,12 +51,9 @@ static cl::opt<cl::boolOrDefault>
     EnableGlobalMerge("riscv-enable-global-merge", cl::Hidden,
                       cl::desc("Enable the global merge pass"));
 
-static cl::opt<bool> ForceEnableGlobalMergeExternalGlobals(
-    "riscv-force-enable-global-merge-external-globals", cl::Hidden,
-    cl::init(false),
-    cl::desc(
-        "If the global merge pass is enabled, force enable global merging of "
-        "external globals (overriding any logic that might disable it)"));
+static cl::opt<bool> DisableGlobalMergeExternalGlobals(
```

Comment:
Is this option redundant with the `global-merge-on-external` option in the pass?

Reply:
Thank you, it is - the naming "MergeExternalByDefault" should have made me go check. I've now deleted and updated the test that relies on this - thank you!